lib/AST/ASTMangler.cpp

```diff
@@ -540,8 +540,15 @@ void ASTMangler::appendDeclName(const ValueDecl *decl) {
         break;
     }
   } else if (decl->hasName()) {
-    // TODO: Handle special names
-    appendIdentifier(decl->getBaseName().getIdentifier().str());
+    // FIXME: Should a mangled subscript name contain the string "subscript"?
```

Comment:
Something @eeckstein may have an opinion on, but can be changed in a follow-up commit.